Project.toml

```diff
@@ -18,8 +18,9 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-POMDPSimulators = "< 0.3.0"
-julia = "1"
+POMDPSimulators = "0.3.0"
```

Comment:
This line should be `POMDPSimulators = "0.3"`. Does the difference make sense?

Reply:
Ah, so it subsumes any 0.3.x versions?